api/views.py

```diff
@@ -234,6 +234,38 @@ def get_statistics(request):
 
     return Response(data=list(result))
 
+@api_view(["POST"])
+@permission_classes((permissions.IsAuthenticated,))
+def get_user_statistics(request):
+    user = request.user
+    body = json.loads(request.body)
+    game, start, end = (
+        body["game"],
+        body["start"],
+        body["end"],
+    )
+
+    if not game or not start or not end:
+        return HttpResponseBadRequest(
+            "Body must contains 'game', 'start' and 'end'"
+        )
+
+    objects = None
+    if game == "memory":
```

Comment:
can we use enum here? I believe enum is already defined in models or helpers

Reply:
I don't think we have a proper enum for this yet. We should also change it in other places. I will create an issue to do this later everywhere.